src/components/Pivot/Pivot.scss

```diff
@@ -77,12 +84,14 @@
   }
 }
 
-
-
+.ms-Fabric.is-focusVisible .ms-Pivot-link {
+  &:focus {
+    outline: 1px solid $ms-color-neutralSecondaryAlt;
```

Comment:
Just checking--does this clash at all with the implementation below for the tabs component?